app-backend/datamodel/src/main/scala/Scope.scala

```diff
@@ -505,4 +519,29 @@ object Scopes {
           PlatformDomainAdminScope
         )
       )
+
+  case object GroundworkUser
```

Comment:
This was the least verbose I could come up with. It should match what is [here](https://docs.google.com/spreadsheets/d/1CD1dK47UpVcqgBsl3d9Ol0Fk5po9LKmKnTNXZbPHcDg/edit#gid=115836025)